### ISSUE TITLE
implement accessibility-friendly way to hide <h1> on homepage

### ISSUE THREAD

**Issue by sahiti-enjeti** [gwu-libraries/lai-drupal#12]
Currently, the h1 tag is being hidden via `display:none` in css for blind/low-vision students. Screen readers are unable to access the h1 tag. Therefore, blind/low-vision students accessing the academic commons page via a screen reader, should be able to hear the text of the h1 tag.

**Comment by maxstarkenburg** [comment]
Closing this here since it's being fixed via the academiccommons-theme repo.
